client/src/components/InputTextField/InputTextField.css

```diff
@@ -0,0 +1,41 @@
+.input-text-field-container input {
+  display: flex;
+  color: black;
+  width: 455px;
+  height: 55px;
+  background-color: #fff;
```

Comment:
Is there a variable for the color that you can use?

Reply:
There should be. Pure white or black isn't actually used. So, the variables for black or white will work in their place.